DrugResistance/src/main/java/edu/stanford/hivdb/drugresistance/GeneDRAsi.java

```diff
@@ -16,19 +16,19 @@
     along with this program.  If not, see <http://www.gnu.org/licenses/>.
 */
 
-package edu.stanford.hivdb.drugresistance;
```

Comment:
I commented out this class because my understanding is that GeneDRFast succeeded GeneDRAsi.

Reply:
Sounds good. I've implemented your recommendation, and I will confirm that it worked on my next commit.